Services/CO.CDP.Forms.WebApi/UseCase/UpdateFormSectionAnswersUseCase.cs

```diff
@@ -23,8 +23,37 @@ public async Task<bool> Execute((Guid formId, Guid sectionId, Guid answerSetId,
         var section = await formRepository.GetSectionAsync(formId, sectionId)
             ?? throw new UnknownSectionException($"Unknown section {sectionId} in form {formId}.");
 
-        var sharedConsent = await formRepository.GetSharedConsentDraftWithAnswersAsync(formId, organisationId)
-                            ?? CreateSharedConsent(organisation, section.Form);
+        var sharedConsent = await formRepository.GetUntrackedSharedConsent(formId, organisationId);
+        if (sharedConsent?.SubmissionState == Persistence.SubmissionState.Submitted)
+        {
+            CleanSharedConsent(sharedConsent);
```

Comment:
Aren't we suppose to create a new SharedConsent and Copy existing Answer & AnswerSet (excluding Declaration items) here?

Reply:
done.